src/lib/y2partitioner/dialogs/blk_device_resize.rb

```diff
@@ -188,28 +211,35 @@ def help
         end
 
         # @macro seeAbstractWidget
-        # Whether the indicated value is valid. It must be a disk size
-        # between min and max possible sizes.
+        # Whether the given size is valid. It must be a size between the
+        # min and max possible sizes.
         #
         # @note An error popup is shown when the given size is not valid.
+        #   A warning popup is shown if there are some warnings.
         #
-        # @return [Boolean] true if the given size is valid; false otherwise.
+        # @see #errors
+        # @see #validation_warnings
+        #
+        # @return [Boolean] true if the given size is valid and the user
```

Comment:
Well, it is tricky. In case of no errors, the size is valid, but there could be warnings. In that case, the user is asked. So, I would say that the method returns true ""if there are no errors and the user accepts the warnings (if someone)". I will improve it.